.travis.yml

```diff
@@ -45,14 +45,17 @@ matrix:
     - name: "c: linux gcc"
       language: cpp
       compiler: gcc
+      env:      PREFIX=/tmp/icu-prefix
       before_script:
         - mkdir build
         - cd build
-        - ../icu4c/source/runConfigureICU --enable-debug --disable-release Linux
+        - ../icu4c/source/runConfigureICU --enable-debug --disable-release Linux --prefix="${PREFIX}"
         - make -j2
       script:
         - make -j2 check
         - ( cd ../icu4c/source/test/depstest && ./depstest.py ../../../../build/ )
+        - make install
```

Comment:
I suppose you could cd to `test/hdrtst` first, then just do `make check`, but I don't think it really matters.

Reply:
The test/hdrtest is testing installed headers, not in-tree headers, so we need to install the package and then test.

Reply:
Sorry, I was confused.